src/components/molecules/ProfilePictureInput/ProfilePictureInput.tsx

```diff
@@ -110,21 +128,23 @@ const ProfilePictureInput: React.FunctionComponent<PropsType> = ({
       {error && <small>Error uploading: {error}</small>}
       <small>Or pick one from our Sparkle profile pics</small>
       <div className="default-avatars-container">
-        {sparkleAvatars.map((avatar, index) => {
-          return (
-            <div
-              key={`${avatar}-${index}`}
-              className="profile-picture-preview-container"
-              onClick={() => uploadDefaultAvatar(avatar)}
-            >
-              <img
-                src={`/avatars/${avatar}`}
-                className="profile-icon profile-picture-preview"
-                alt="your profile"
-              />
-            </div>
-          );
-        })}
+        {isLoadingAvatars && <div>Loading...</div>}
```

Comment:
Do we want to mark just this part as loading, or the whole component? Usually we have done the whole component from what I have seen?

Reply:
On on side, I can load the whole component and keep it consistent. On the other, whilst the avatars are loading, the user can still type his display name and see the rest of the content on the screen. My idea was not to block his whole page just because the avatars are loading.

Reply:
Mm, makes sense. Let's not block the whole page.